inc/thorvg.h

```diff
@@ -756,6 +756,7 @@ class TVG_EXPORT Shape final : public Paint
      *
      * The current point value is set to the last point from the sub-path.
      * For each command from the @p cmds array, an appropriate number of points in @p pts array should be specified.
+     * If the number of points array is different from the number required by commands array, the shape with this path added is not displayed on the screen.
```

Comment:
Hi @mgrudzinska 
appendPath() stores all the passed path informations. 
Due to the change in https://github.com/Samsung/thorvg/pull/738, the behavior has been changed so that the shape is not output if the pair(cmds, points) does not match.
Thank you fore check :)